### ISSUE TITLE
Err encoding label

### ISSUE THREAD

**Issue by phamkhactu** [NielsRogge/Transformers-Tutorials#184]
@NielsRogge thanks for great sharing to training TrOOC, I begin step by step as you [guide](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Fine_tune_TrOCR_on_IAM_Handwriting_Database_using_native_PyTorch.ipynb), but I get error that:
My label is "Khánh Nguyên" but after encode from code, the result is: "Trung"
```
encoding = train_dataset[0]
for k,v in encoding.items():
  print(k, v.shape)
labels = encoding['labels']
labels[labels == -100] = processor.tokenizer.pad_token_id
label_str = processor.decode(labels, skip_special_tokens=True)
print(label_str)
```
Here is my tokenizer:

```
tokenizer = AutoTokenizer.from_pretrained("NlpHUST/vibert4news-base-cased")
processor = TrOCRProcessor.from_pretrained("microsoft/trocr-base-handwritten")
processor.tokenizer = tokenizer
```
Thank you
